freeipa/src/main/java/com/sequenceiq/freeipa/configuration/CloudIdSyncConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.freeipa.configuration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CloudIdSyncConfig {
+
+    @Value("${freeipa.cloudidsync.poller.timeoutMs:4000}")
```

Comment:
Can you please remove the default values from here? The application.yml already contains them.